docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,459 @@
+Using Public Clouds as backends
+===============================
+
+Introduction
+------------
+
+As stated in our `GETTING STARTED guide <../GETTING_STARTED/#location-configuration>`__,
+new backends can be added by creating a region (also called location constraint)
+with the right endpoint and credentials.
+This section of the documentation shows you how to set up our currently
+supported public cloud backends:
+
+- `Amazon S3 <#aws-s3-as-a-backend>`__ ;
+- `Microsoft Azure <#microsoft-azure-as-a-backend>`__ .
+
+For all Public Cloud backends, you'll have to do some editing inside the
+repository, and some editing from the Public Cloud Console.
+
+.. IMPORTANT::
+   CloudServer also enables you to have, within the same local bucket, objects
+   stored to different backends (both private and public clouds, for example).
+   This is an advanced usecase, and it will require that you set your backends
+   as explained below. Once this is done, you may refer to the
+   `backend configuration at object level <#backend-configuration-at-object-level>`__
+   section of this document.
+
+AWS S3 as a backend
+-------------------
+
+From the AWS S3 Console
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a bucket where you will host your data for this new location constraint.
+This bucket must have versioning enabled. This is an option you may choose to
+activate at step 2 of Bucket Creation in the console.
```

Comment:
the aws console? Perhaps, just instruct them to set up an AWS bucket with their client of choice and enable versioning?

Reply:
I added pointers for AWS CLI as well as the Console info; from what I see at events, ppl either use the Console or AWS CLI, so it might be user friendly to give pointers for these two tools.